src/main/java/de/zalando/aruha/nakadi/config/SecurityConfiguration.java

```diff
@@ -58,11 +58,13 @@ public void configure(final HttpSecurity http) throws Exception {
                     .antMatchers(POST, "/event-types/**").access(hasScope(eventTypeWriteScope))
                     .antMatchers(PUT, "/event-types/**").access(hasScope(eventTypeWriteScope))
                     .antMatchers(GET, "/health/**").permitAll()
+                    .antMatchers(GET, "/version/**").permitAll()
```

Comment:
Sorry, forgot to mention that when first looking at pull request. As you added an endpoint and defined security rules for it - please extend authentication tests to cover that.
https://github.com/zalando/nakadi/blob/nakadi-jvm/src/test/java/de/zalando/aruha/nakadi/security/BasicModeAuthenticationTest.java
https://github.com/zalando/nakadi/blob/nakadi-jvm/src/test/java/de/zalando/aruha/nakadi/security/FullModeAuthenticationTest.java
https://github.com/zalando/nakadi/blob/nakadi-jvm/src/test/java/de/zalando/aruha/nakadi/security/OffModeAuthenticationTest.java